src/Saritasa.Tools.Messages/Properties/AssemblyInfo.cs

```diff
@@ -17,3 +17,5 @@
 
 [assembly: AssemblyVersion("0.1.0.0")]
 [assembly: AssemblyFileVersion("0.1.0.0")]
+
+[assembly: InternalsVisibleTo("Saritasa.Tools.Messages.Tests")]
```

Comment:
Please add `PublicKey=...` for the friendly assembly in order to internal classes could be visible for the Tests project.